PluralKit.Bot/Commands/MemberCommands.cs

```diff
@@ -58,6 +58,21 @@ public MemberCommands(IDataStore data, EmbedService embeds, ProxyCacheService pr
             await _proxyCache.InvalidateResultsForSystem(ctx.System);
         }
         
+        public async Task MemberRandom(Context ctx)
+        {
+            ctx.CheckSystem();
+
+            var randGen = new System.Random(); 
+            //Maybe move this somewhere else in the file structure since it doesn't need to get created at every command
+
+            var members = await _data.GetSystemMembers(ctx.System);
```

Comment:
GetSystemMembers returns a `IEnumerable`, so you need to wrap it in parens and then call `.ToList()`.